easybuild/easyconfigs/r/R/R-4.0.0-foss-2020a.eb

```diff
@@ -2452,60 +2452,6 @@ exts_list = [
     ('drugCombo', '1.1.1', {
         'checksums': ['9fdc3a7cf63552c32f1c7573258fc4ceacdaf5c475fe79aa4ca8c9226b9f8a38'],
     }),
-    ('betareg', '3.1-3', {
-        'checksums': ['cc19387ec516492d11cf59cdfa07e1733950a2af8196c1e155bc95939bc76246'],
-    }),
-    ('unmarked', '1.0.1', {
-        'checksums': ['a3bb9bdc7a4a79ea38482df3f8cbb6e9082332a0d894eeb4b3dc816344cec0e4'],
-    }),
-    ('maxlike', '0.1-8', {
-        'checksums': ['90aaab9602f259cbfae61fe96e105cc4a0c2a385b42380f85c14f5d544107251'],
-    }),
-    ('coxme', '2.2-16', {
-        'checksums': ['a0ce4b5649c4c1abbfe2c2bf23089744d1f66eb8368dea16e74e090f366a5111'],
-    }),
-    ('AICcmodavg', '2.3-0', {
-        'checksums': ['4d6bcff3c549be9dcefdd849b239659618fdaf9ba0d27b9d0589620d104e5e24'],
-    }),
-    ('pacman', '0.5.1', {
-        'checksums': ['9ec9a72a15eda5b8f727adc877a07c4b36f8372fe7ed80a1bc6c2068dab3ef7c'],
-    }),
-    ('spaa', '0.2.2', {
-        'checksums': ['a5a54454d4a7af473ce797875f849bd893005cb04325bf3e0dbddb19fe8d7198'],
-    }),
-    ('maxnet', '0.1.2', {
-        'checksums': ['dfa02ca1031b369415d8b16863ca5fd115c7bf96b1f8fc24f91719b017f5cce5'],
-    }),
-    ('ENMeval', '0.3.0', {
-        'checksums': ['1c924098a27c82a9bf75408173b47429c40cc41cae3aba18c86ba217bb45ba60'],
-    }),
-    ('plotmo', '3.5.7', {
-        'checksums': ['aa366a49a63cabfd5d799a1524e23a4faed022f10ee60f8407d70ab70731b38c'],
-    }),
-    ('earth', '5.1.2', {
-        'checksums': ['326f98e8c29365ca3cd5584cf2bd6529358f5ef81664cbd494162f92b6c3488d'],
-    }),
-    ('mda', '0.5-2', {
-        'checksums': ['344f2053215ddf535d1554b4539e9b09067dac878887cc3eb995cef421fc00c3'],
-    }),
-    ('biomod2', '3.4.6', {
-        'checksums': ['41fd7745f4d0af3f799e9cf4fa5484a427de6854db84c6476fde7a7414787d5b'],
-    }),
-    ('poLCA', '1.4.1', {
-        'checksums': ['2e69975b5e7da8c36641bfa9453afdb4861523866b8799bec1d4eace9ab5762e'],
-    }),
-    ('PermAlgo', '1.1', {
-        'checksums': ['d7157b92241c34b71ad19901b52144973b49df453bf2a5edf4497d4bf26bd099'],
-    }),
-    ('coxed', '0.3.3', {
-        'checksums': ['d0d6cb8fea9516b3c63b34d0d81f3804c18a07f97a83e51555575c8ed4c75626'],
-    }),
-    ('testit', '0.11', {
-        'checksums': ['12fb24f0c7b032ae0d146d22161efc676ef4cdf604b81f6345e8d9dfd125388e'],
-    }),
-    ('data.tree', '1.0.0', {
-        'checksums': ['40674c90a5bd00f5185db9adbd221c6f1114043e69095249f5fa8b3044af3f5e'],
-    }),
```

Comment:
Ah, now I understand your comment..

Yeah, please don't remove this stuff which is already in `develop`. :)

Reply:
I managed to make the changes against the 4.2.2 branch by mistake, and then I ran an overnight test of the synced eb against develop to make sure it built. Pushed now.